v2/sourcedb-to-spanner/README.md

```diff
@@ -87,3 +87,23 @@ gcloud dataflow flex-template run ${JOB_NAME} \
         --parameters sourceConfigURL="jdbc:mysql://<source_ip>:3306/<mysql_db_name>",username=<mysql user>,password=<mysql pass>,instanceId="<spanner instanceid>",databaseId="<spanner_database_id>",projectId="$PROJECT",outputDirectory=gs://<gcs-dir> \
         --additional-experiments=disable_runner_v2
 ```
+#### Replaying DLQ entries.
+Any errors to transform a source row or failures to write to spanner get written to `dlq/severe/` path within the `outputDirectory`. It's recommended to retry the DLQ entries before applying any change capture (if any).
+To retry the DLQs, please run the `Cloud_Datastream_to_Spanner` job in `retryDLQ` mode. After the DLQs are successfully applied, the files will be deleted from the dlq directory.
+##### Sample Command to retry DLQs.
+The following sample command could help to start a DLQ retry job.
+```bash
+gcloud  dataflow flex-template run <jobname> \
+--region=<the region where the dataflow job must run> \
+--template-file-gcs-location=gs://dataflow-templates/latest/flex/Cloud_Datastream_to_Spanner \
+--additional-experiments=use_runner_v2 \
+--parameters inputFilePattern=<GCS location of the input file pattern>,streamName="ignore", \
+--datastreamSourceType=<source_type for example mysql/oracle. This needs to be set in the absence of an actual datastream.>\
+instanceId=<Spanner Instance Id>,databaseId=<Spanner Database Id>,sessionFilePath=<GCS path to session file>, \
+deadLetterQueueDirectory=<outputDirectory/dlq>,runMode="retryDLQ"
+```
```

Comment:
Please also link the readme of the live template here for flag reference etc.

Reply:
Makes sense. Added in the recent push. Thanks.